### PR TITLE
Add 0.16.0 manifests of knative

### PR DESCRIPTION
**Pull request by houshengbo**
## Proposed Changes

* This PR added the released 0.16.0 knative.

### CODE REVIEW

File: test/resources/knativeresources.go [hunk 1]
```diff
@@ -177,10 +169,21 @@ func IsKnativeObsoleteResourceGone(clients *test.Clients, namespace string, obsR
 		gvr := apis.KindToResource(resource.GroupVersionKind())
 		var err error
 		if resource.GetNamespace() != "" {
-			// This is a namespaced resource
+			// Verify all namespaced resources, except jobs.
```
Comment (Cynocracy) [inline test/resources/knativeresources.go:172]: Why not Jobs?

File: test/resources/knativeresources.go [hunk 2]
```diff
@@ -177,10 +169,21 @@ func IsKnativeObsoleteResourceGone(clients *test.Clients, namespace string, obsR
...
+			}
 			_, err = clients.Dynamic.Resource(gvr).Namespace(namespace).Get(resource.GetName(), metav1.GetOptions{})
 		} else {
-			// This is a clustered resource
+			// TODO(#1): If APIVersion is the only different field between two resources with
+			// one being v1 and the other being v1beta1, the dynamic client can access both of
+			// them in the cluster. Before we find out the reason, we skip verifying CRDs and
+			// webhooks for all clustered resources.
+			switch strings.ToLower(resource.GetKind()) {
+			case "customresourcedefinition", "validatingwebhookconfiguration", "mutatingwebhookconfiguration":
```
Comment (Cynocracy) [inline test/resources/knativeresources.go:184]: I'd be more OK with this if we're confident we're going to get rid of this for job upgrades later.

Are we definitely moving forward with that? Can we add a TODO to clean this up?

File: test/resources/knativeresources.go (hunk 1 above)
Comment (houshengbo) [reply test/resources/knativeresources.go:172]: We added a job transformation, so the job name won't match the one of the launched job. It is meaningless to check jobs.

Comment (Cynocracy) [reply test/resources/knativeresources.go:172]: OK, so long as we don't create a new job every reconcile loop, I suppose.

I wouldn't say it's meaningless to check jobs, though. They may be necessary for the correctness of the upgrade.

**Pull request merged**